test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
```

Comment:
When running this, I'm getting:

```
/Users/sirup/src/github.com/Shopify/kubernetes-deploy/test/test_helper.rb:68:in `create_namespace': uninitialized constant Kubeclient::Namespace (NameError)
```

I had to change `Kubernetes::Namespace` there to `Kubernetes::Resource`. How does this work? 🤔